src/google/protobuf/stubs/stringpiece.h

```diff
@@ -214,6 +214,13 @@ class PROTOBUF_EXPORT StringPiece {
       : ptr_(str.data()), length_(0) {
```

Comment:
I tried to fix this one by replacing const string& with std::string_view, but certain cases were broken.

I. e. HasSuffixString(const string&, const string&) stops working as it requires 2 implicit ctors (and the standard only allows one).

This could be simplified with C++20 starts_with / ends_with, but this is a long story.